team-app/src/Search.js

```diff
@@ -0,0 +1,40 @@
+import { useState } from "react";
+
+export default Search = ({ setFilteredCards, teammates }) => {
+  const [searchInput, setSearchInput] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  const handleSubmit = (event) => {
+    try {
+      event.preventDefault();
+      setErrorMessage("");
+
+      if (searchInput != "" && searchInput.length < 3) {
+        setErrorMessage("Please enter at least 3 letters");
+        return;
+      }
+
+      const filteredMembers = teammates.filter(
+        (member) =>
```

Comment:
Try to keep this reusable function out of the component and import it in the component then